app/build.gradle

```diff
@@ -411,6 +412,7 @@ dependencies {
     implementation Deps.androidx_navigation_fragment
     implementation Deps.androidx_navigation_ui
     implementation Deps.androidx_recyclerview
+    implementation Deps.androidx_lifecycle_livedata
```

Comment:
Why do we need to introduce LiveData for the share sheet? Is it just for caching the app list and separating the connectivity checks? The AccountManager already has a cache for the shareable devices, and the app list is a short-lived in-memory cache while you're in the ShareFragment. We probably move the connectivity checks to separate class as well.

Reply:
We already have LiveData, this just switches the KTX version instead.

I wanted to break out the app and device list code into a separate class for testing, and realized I was just recreating a view model. `LiveData` is a better representation instead of `Deferred` for the data, since the device list can be updated.

I previously checked with the Fenix team about using LiveData and it's fine to use it where appropriate.